ratls/src/lib.rs

```diff
@@ -49,6 +51,14 @@ pub enum TeeType {
     Sev,
 }
 
+pub enum TeeMeasurement {
+    Sgx {
+        mr_signer: [u8; 32],
+        mr_enclave: [u8; 32],
+    },
```

Comment:
It'll force the user to specify both `MRENCLAVE` ansd `MRSIGNER`.

Is what we want?

Reply:
Great question. To be fair, my idea was to update that if needed later. The only usage for now is the KMS and the question is still opened. For now I say: do not check the measurements or check all of them.